app/backend/wells/management/commands/export.py

```diff
@@ -209,8 +210,18 @@ def generate_files(self, zip_filename, spreadsheet_filename):
  registries_person.person_guid = well.person_responsible_guid
  left join registries_organization on
  registries_organization.org_guid = well.org_of_person_responsible_guid
+
+-- WELL LICENCES: any well having at least 1 licence entry will be marked as Licensed.
+    left join (select well_tag_number, count(*) as cur_licences from well
+    join well_licences on
+    well.well_tag_number = well_licences.well_id
+    group by well_tag_number) as licence_q
+    on well.well_tag_number = licence_q.well_tag_number
+
```

Comment:
Maybe this comment can be moved out of this sql string? I like the intention, but it leaves the string vulnerable for modification (i.e. a typo or a change in formatting can invalidate it).